src/RecoveryServices/RecoveryServices.Backup.Autorest/custom/Cmdlets/Policy/New-AzRecoveryServicesBackupPolicy.ps1

```diff
@@ -0,0 +1,86 @@
+
+	param(
+        [Parameter(Mandatory=$false, HelpMessage='Subscription Id')]
+        [System.String]
+        ${SubscriptionId},
+
+        [Parameter(Mandatory, HelpMessage='The name of the resource group where the recovery services vault is present.')]
+        [System.String]
+        ${ResourceGroupName},
+
+        [Parameter(Mandatory, HelpMessage='The name of the recovery services vault.')]
+        [System.String]
+        ${VaultName},
+
+        [Parameter(Mandatory, HelpMessage='Policy Name for the policy to be created')]
+        [System.String]
+        ${PolicyName},
+
+        [Parameter(Mandatory, HelpMessage='Workload specific Backup policy object.')]
+        [Microsoft.Azure.PowerShell.Cmdlets.RecoveryServices.Models.Api20230201.IProtectionPolicy]
+        ${Policy},
+    
+        [Parameter()]
+        [Alias('AzureRMContext', 'AzureCredential')]
+        [ValidateNotNull()]
+        [System.Management.Automation.PSObject]
+        # The credentials, account, tenant, and subscription used for communication with Azure.
+        ${DefaultProfile},
+    
+        [Parameter(DontShow)]
+        [System.Management.Automation.SwitchParameter]
+        # Wait for .NET debugger to attach
+        ${Break},
+    
+        [Parameter(DontShow)]
+        [ValidateNotNull()]
+        [Microsoft.Azure.PowerShell.Cmdlets.RecoveryServices.Runtime.SendAsyncStep[]]
+        # SendAsync Pipeline Steps to be appended to the front of the pipeline
+        ${HttpPipelineAppend},
+    
+        [Parameter(DontShow)]
+        [ValidateNotNull()]
+        [Microsoft.Azure.PowerShell.Cmdlets.RecoveryServices.Runtime.SendAsyncStep[]]
+        # SendAsync Pipeline Steps to be prepended to the front of the pipeline
+        ${HttpPipelinePrepend},
+    
+        [Parameter(DontShow)]
+        [System.Uri]
+        # The URI for the proxy server to use
+        ${Proxy},
+    
+        [Parameter(DontShow)]
+        [ValidateNotNull()]
+        [System.Management.Automation.PSCredential]
+        # Credentials for a proxy server to use for the remote call
+        ${ProxyCredential},
+    
+        [Parameter(DontShow)]
+        [System.Management.Automation.SwitchParameter]
+        # Use the default credentials for the proxy
+        ${ProxyUseDefaultCredentials}
+    )
+
+    process
+    {
+        # RsvRef
+        # public string[] ResourceGuardOperationRequest --- this should be a parameter (check in SDK code) ? If yes, optional parameters ? 
+                
+        # RsvRef : add policy validation (preferably one entry point)
+        
+        $policyObject = [Microsoft.Azure.PowerShell.Cmdlets.RecoveryServices.Models.Api20230201.ProtectionPolicyResource]::new()
+        $policyObject.Property = $Policy
+
+        
+        $null = $PSBoundParameters.Remove("Policy")
+        $null = $PSBoundParameters.Add("Parameter", $policyObject)
```

Comment:
Autogenerated command has the parameter with name **Parameter** and we are currently hiding the auto generated command. So this seems to be correct as is. do you still want us to modify the parameter "Parameter" in the hidden auto generated command ?

Also, in the autogenerated command the type is slightly different, we are defining $policyObject with right type. This looks correct to me. Pls let me know if you think otherwise.